services/meta/data.go

```diff
@@ -137,40 +137,31 @@ func (data *Data) RetentionPolicy(database, name string) (*RetentionPolicyInfo,
 // Returns an error if name is blank or if a database does not exist.
 func (data *Data) CreateRetentionPolicy(database string, rpi *RetentionPolicyInfo) error {
 	// Validate retention policy.
-	if rpi.ReplicaN < 1 {
+	if rpi.Name == "" {
```

Comment:
This should never be passed a blank retention policy. I can add it anyway though.